firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -155,7 +156,7 @@ int stillness_timer_set(String input) {
 
     // if e, echo the current threshold
     if (*holder == 'e') {
-        EEPROM.get(ADDR_STATE3_MAX_STILLNES_TIME, state3_max_stillness_time);
+        EEPROM.get(ADDR_STATE3_MAX_STILLNESS_TIME, state3_max_stillness_time);
```

Comment:
Fixing the typo in this constant name to avoid future headaches.

Reply:
Nice catch!!